Server/Admin/main.py

```
@@ -413,27 +437,28 @@ def get_health_check_results():
-    test_logged = log_request("admin", 200, "/test-rpm", 10.0)
-    
-    # Get current stats
-    stats = await get_stats("admin")
-    
+    """Test endpoint to verify RPM tracking - disabled (redis_logger removed)"""
     return {
-        "message": "RPM tracking test",
-        "test_request_logged": test_logged,
+        "message": "RPM tracking disabled - redis_logger removed",
+        "test_request_logged": False,
         "current_stats": stats,
```

Ruff (0.14.13)

Undefined name `stats`

(F821)